### PR TITLE
Re-arrange onboarding steps

### PR DESCRIPTION
**Pull request by bfollington**
Fixes https://github.com/subconsciousnetwork/subconscious/issues/910

> (Informational) Welcome to subconscious [Next]
> (Informational) "Your sphere stores all your data" [Create sphere]
> (Sphere created) Show mnemonic [I've written it down]
> Nickname [Next]
> Invite code [Redeem]

# Changes
- Extract invite code form to new `FirstRunInviteView` step
- Re-order steps based on above description
- Wait to `.createSphere` until the `FIrstRunSphereView` step is submitted
- Introduce a new visualization `FirstRunOrbitEffectView`
- Normalize spacing between all "FirstRun" type screens
- Reduce jank when showing/hiding logo based on keyboard focus

# TODO
- [x] Tidy code in `FirstRunOrbitEffectView`
- [ ] Finalize copy

# Demo

https://github.com/subconsciousnetwork/subconscious/assets/5009316/a19e58e8-81f9-4efc-a4ea-8ef37dc02e94

### CODE REVIEW

File: xcode/Subconscious/Shared/Components/AppUpgradeView.swift [hunk 1]
```diff
@@ -49,13 +49,13 @@ struct AppUpgradeView: View {
...
-                Spacer()
             }
             .foregroundColor(.primary)
             .multilineTextAlignment(.center)
-            .font(.title3)
+            .font(.body)
             .bold()
-            .frame(height: AppTheme.unit * 32)
+            
+            Spacer()
```
Comment (bfollington) [inline xcode/Subconscious/Shared/Components/AppUpgradeView.swift:58]: Matching the layout of the FirstRun view

File: xcode/Subconscious/Shared/Components/AppUpgradeView.swift [hunk 2]
```diff
@@ -104,7 +104,12 @@ struct AppUpgradeView: View {
             .buttonStyle(PillButtonStyle())
             .disabled(!state.isComplete)
         }
-        .padding()
+        .padding(EdgeInsets(
+            top: AppTheme.unit2,
+            leading: AppTheme.padding,
+            bottom: AppTheme.padding,
+            trailing: AppTheme.padding
+        ))
```
Comment (bfollington) [inline xcode/Subconscious/Shared/Components/AppUpgradeView.swift:112]: Matching the layout of the FirstRun view

File: xcode/Subconscious/Shared/Components/FirstRun/FirstRunOrbitEffectView.swift [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        }
+    }
+    
+    var outerRing: [GenerativeProfilePic] {
+        [Int](
+            repeating: 0,
+            count: 15
+        ).map { _ in
+            GenerativeProfilePic(did: Did.dummyData(), size: 20)
+        }
```
Comment (bfollington) [inline xcode/Subconscious/Shared/Components/FirstRun/FirstRunOrbitEffectView.swift:70]: I wanted to do:

```swift
[GenerativeProfilePic](
    repeating: GenerativeProfilePic(did: Did.dummyData(), size: 20),
    count: 15
)
```

But I need the `repeating:` parameter to be a new instance each time.

**Review by gordonbrander — CHANGES_REQUESTED** [review]
Playtested on-device. Looks and feels great.

I would like to make the following adjustments to the copy... other than that, LGTM!

### Welcome to Subconscious

Subconscious is a place to garden thoughts and share them with others.

It’s powered by Noosphere, a decentralized protocol, so your data belongs to you.

### Create Sphere

Subconscious stores your notes in your personal sphere.

Your sphere is stored on your device, and connects to other spheres over a decentralized network.

### Recovery Phrase

This is your sphere's secret recovery phrase. You can use it to recover your data if you lose access.

It's for your eyes only. We don't store it. Write it down or add it to your password manager.

### Your Profile

Give your sphere a nickname:

### Invite Code

Get connected to Noosphere.

Enter your invite code to create your gateway.

### Done

Creating your gateway...

You can start exploring offline.

_or_

Connected!

Welcome to Subconscious.

**Comment by bfollington** [comment]
@gordonbrander nice, I like it. Updated & re-requested review.

**Review by gordonbrander — APPROVED** [review]
LGTM

**Pull request merged**